app/exec/workitem-create.ts

```diff
@@ -0,0 +1,90 @@
+import cmdm = require('../lib/tfcommand');
+import cm = require('../lib/common');
+import witifm = require('vso-node-api/interfaces/WorkItemTrackingInterfaces');
+import VSSInterfaces = require('vso-node-api/interfaces/common/VSSInterfaces');
+import witm = require('vso-node-api/WorkItemTrackingApi');
+import argm = require('../lib/arguments');
+var trace = require('../lib/trace');
+
+export function describe(): string {
+    return 'create a workitem';
+}
+
+export function getCommand(): cmdm.TfCommand {
+    // this just offers description for help and to offer sub commands
+    return new WorkItemCreate;
+}
+
+// requires auth, connect etc...
+export var isServerOperation: boolean = true;
+
+// unless you have a good reason, should not hide
+export var hideBanner: boolean = false;
+
+export class WorkItemCreate extends cmdm.TfCommand {
+    
+    requiredArguments = [argm.PROJECT_NAME, argm.WORKITEMTYPE, argm.TITLE];
+    optionalArguments = [argm.ASSIGNEDTO, argm.DESCRIPTION];
+    
+    public exec(args: string[], options: cm.IOptions): any {
+        trace('workitem-create.exec');
+        var witapi: witm.IQWorkItemTrackingApi = this.getWebApi().getQWorkItemTrackingApi();
+        
+        return this.promptArguments(this.requiredArguments, this.optionalArguments).then( (allArguments) => {
```

Comment:
Also - just to make sure - if I supply the args on the cmd line, it won't prompt correct?  (that was one of the initial goals)